src/threat_model/threat_model.md

```diff
@@ -312,16 +312,16 @@ cleared.
-| 2  | The authenticators should only allow authentication tokens that are secure enough that brute-force attacks are not feasible.                                                                                                                                                                                                                                                          |
-| 3  | Listener implementations use an IPC mechanism respecting confidentiality and integrity of messages transmitted between the clients and the service (once the initial connection has been made). - Unix Domain Socket Listener: the sockets used on the client and service side for the communication are represented by file descriptors that are only accessible by those processes. |
-| 4  | The following events are logged by the service with information containing the application identity, time and date: "new request received", "response sent back", "new mapping created", "mapping is read from the persistent store", "a hardware command is issued" and "a hardware command is received".                                                                            |
-| 5  | Parsec is coded with safety in mind and is tested extensively (TODO: add a link of test document describing the tests that we do in high-level terms on what platform).                                                                                                                                                                                                               |
-| 6  | The `ManageKeyInfo` implementations communicate with its persistence storage backend through a mechanism respecting confidentiality, integrity and availability. On-Disk Key Info Manager: the mappings are stored on-disk using the filesystem provided by the OS. OS file permissions are used.                                                                                     |
-| 7  | Logging is configured to include the software component from where logs are generated.                                                                                                                                                                                                                                                                                                |
+| ID | Justification                                                                                                                                                                                                                                                                                              | Status                                                                                                                                                                              |
```

Comment:
I'd probably name the second column "details", as for some of them (e.g. M-3) it's not really a status

Reply:
Done